src/views/v2/list/event/venue.php

```diff
@@ -31,8 +31,13 @@
 	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
 		<?php echo esc_html( $address ); ?>
+
+		<?php if ( $venue->city ) : ?>
```

Comment:
check if empty rather than just checking venue city

Reply:
I've updated this to use if ! empty